keyboards/leopold/fc980c/wlhlm/keymaps/default/keymap.c

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 Wilhelm Schuster
```

Comment:
Since the main folder has a default keymap, not sure that this one needs it,

Reply:
I'm sorry, I don't quite understand. Do you mean the default keymap in fc980c/tmk?